packages/react-compose/src/types.ts

```diff
@@ -34,7 +34,7 @@ export type ComponentWithAs<TElementType extends keyof JSX.IntrinsicElements = '
 
   /**
    * A hack to simplify the resolution for ComponentWithAs.
-   * @see https://github.com/microsoft/fluentui/pull/13841
+   * See https://github.com/microsoft/fluentui/pull/13841
```

Comment:
This might have been needed due to the new API Extractor version complaining about unrecognized tags